entries/b/bitwarden.com.json

```diff
@@ -11,9 +11,9 @@
       "custom-software"
     ],
     "documentation": "https://help.bitwarden.com/article/setup-two-step-login/",
-    "notes": "Premium account required for Duo, YubiKey and FIDO U2F.",
+    "notes": "Premium account required for Duo, YubiKey and FIDO U2F. Teams or Enterprise account required for SMS and phone call",
```

Comment:
"Premium" accounts also allow for SMS and phone call options (via _Duo_), if I'm not mistaken.

Reply:
No they don't I checked. I have a premium account and don't have access to those methods

Reply:
Correct, @phallobst. See [Two-step Login Methods](https://bitwarden.com/help/article/setup-two-step-login/).

Reply:
Something like 2FA options offered differ depending on subscription plan. Would probably be sufficient as a note.